src/neuroglancer/image_user_layer.ts

```diff
@@ -84,6 +86,7 @@ export class ImageUserLayer extends Base {
       channelCoordinateSpace => makeValueOrError(() => getChannelSpace(channelCoordinateSpace)),
       this.channelCoordinateSpace));
   volumeRendering = new TrackableBoolean(false, false);
```

Comment:
I think it would make more sense to just use the existing `volumeRendering` variable but change its type from `boolean` to enum.  There can be a NONE value or similar to indicate it is disabled.